Sources/InstanceStorage.swift

```diff
@@ -120,7 +120,10 @@ private class Weak<Wrapped> {
     }
 #else
     var value: Wrapped? {
-        get { return object as? Wrapped }
+        get {
+            guard let object = object else { return nil }
+            return object as? Wrapped
+        }
```

Comment:
This fix also needs to be applied to the linux branch

Reply:
Adjusted in ce3b62e702d8fb59a41999353c8a257878adc035

Reply:
👍